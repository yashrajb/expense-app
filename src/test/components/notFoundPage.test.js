import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage.js';

test('should render not found page ',() => {

    const result = shallow(<NotFoundPage />);
    expect(result).toMatchSnapshot();

});