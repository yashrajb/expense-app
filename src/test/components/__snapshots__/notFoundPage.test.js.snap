// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page  1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
